<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="fs1Layout"/>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="fs2Layout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>ACD</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionShowIcons"/>
    <addaction name="actionShowHidden"/>
    <addaction name="actionShowSecondBrowser"/>
    <addaction name="separator"/>
    <addaction name="actionNextFile"/>
    <addaction name="actionPreviousFile"/>
    <addaction name="separator"/>
    <addaction name="actionRenameFiles"/>
    <addaction name="actionCopyFiles"/>
    <addaction name="actionMoveFiles"/>
    <addaction name="actionDeleteFiles"/>
    <addaction name="actionMkDir"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionShowIcons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Icons</string>
   </property>
  </action>
  <action name="actionShowHidden">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Hidden</string>
   </property>
  </action>
  <action name="actionShowSecondBrowser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 2nd file browser</string>
   </property>
  </action>
  <action name="actionCopyFiles">
   <property name="text">
    <string>Copy File(s)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionMoveFiles">
   <property name="text">
    <string>Move File(s)</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionDeleteFiles">
   <property name="text">
    <string>Delete File(s)</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionRenameFiles">
   <property name="text">
    <string>Rename File(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionMkDir">
   <property name="text">
    <string>Create Directory</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionNextFile">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPreviousFile">
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
